docs/sources/setup-grafana/configure-security/configure-scim-provisioning/manage-users-teams/_index.md

```diff
@@ -0,0 +1,160 @@
+   - Grafana expects the SAML integration to return the same External ID in SAML assertions
+   - This External ID is used to validate that the logged-in user matches the provisioned user
+
+This process ensures secure and consistent user identification across both systems, preventing security issues that could arise from email changes or other user attribute modifications.
+
+### Existing Grafana users
+
+{{< admonition type="note" >}}
+Existing users must be assigned to the Grafana app in the identity provider to maintain access once SCIM is enabled.
+{{< /admonition >}}
+
+For users who already exist in the Grafana instance:
+
+- SCIM establishes the relationship through the External ID matching process
+- Creates a secure link with the identity provider identity
+- Preserves all existing settings and access
+- Keeps the account active and unchanged until assigned in the identity provider
+
+#### Handling users from other provisioning methods
+
+To prevent conflicts and maintain consistent user management, disable or restrict other provisioning methods when implementing SCIM. This ensures that all new users are created through SCIM and prevents duplicate or conflicting user records.
+
+- SAML Just-in-Time (JIT) provisioning:
+
+  - Disable `allow_sign_up` in SAML settings to prevent automatic user creation
+  - Existing JIT-provisioned users will continue to work but should be migrated to SCIM
+
+- Terraform or API provisioning:
+
+  - Stop creating new users through these methods
+  - Existing users will continue to work but should be migrated to SCIM
+  - Consider removing or archiving Terraform user creation resources
+
+- Manual user creation:
+  - Restrict UI-based user creation to administrators only
+  - Plan to migrate manually created users to SCIM
+
+### New users
+
+For users who don't yet exist in Grafana:
+
+- SCIM creates accounts when users are assigned to Grafana in the identity provider
+- Sets up initial access based on identity provider group memberships and SAML role mapping
+- No manual Grafana account creation needed
+
+### Role management
+
+SCIM handles user synchronization but not role assignments. Role management is handled through [Role Sync](../../configure-authentication/saml#configure-role-sync), and any role changes take effect during user authentication.
+
+## Team provisioning with SCIM
+
+SCIM provides automated team management capabilities that go beyond what Team Sync offers. While Team Sync only maps identity provider groups to existing Grafana teams, SCIM can automatically create and delete teams based on group changes in the identity provider.
+
+For detailed configuration steps specific to the identity provider, see:
+
+- [Configure SCIM with Azure AD](../configure-scim-azure/)
+- [Configure SCIM with Okta](../configure-scim-okta/)
+
+### SCIM vs Team Sync
+
+{{< admonition type="warning" >}}
+Do not enable both SCIM and Team Sync simultaneously as these methods can conflict with each other.
```

Comment:
Ah but I thought we would recommend that:
Coming soon - support for migrating Team Sync to SCIM Group Sync.
We don't recommend migrating from Team Sync until this support is released.
You may want to benefit from SCIM User Sync already, and this is compatible alongside Team Sync.

Reply:
Perhaps what you mean here is: do not enable both team provisioning with SCIM Group Sync?

Reply:
Because currently this reads like, you can't use SCIM at all if you are using Team Sync.

Reply:
>Because currently this reads like, you can't use SCIM at all if you are using Team Sync.

I can see that it is confusing. If you read just this section without looking the entire page, it can create confusion. However, this warning is under the "SCIM vs Team Sync" section, so is only applicable to teams and not users. 

That said, I acknowledge that not everyone will read the doc entirely, so I will change this to make sure it's clear that user sync can still be used.